### ISSUE TITLE
Contact Outred

### ISSUE THREAD

**Issue by Zumwaltboi68** [OutRed/outred-proxy-static#1]
Outred, do you have an email that I can contact you with?

**Comment by Zumwaltboi68** [comment]
Or should I contact you some other way?

**Comment by OutRed** [comment]
Email me at support@outred.org for support/questions.

**Comment by OutRed** [comment]
I see you have a repo called Doge Unblocker, and it has a vercel.json file, even though it uses Ultraviolet. Is it deployable on Vercel?

**Comment by Zumwaltboi68** [comment]
I believe so. It would be great for you to try it out.

**Comment by Zumwaltboi68** [comment]
Do you remember my username from that other time we talked on github? My other account is zumwaltboi.

**Comment by OutRed** [comment]
No. I've talked to a lot of people and don't remember you. I'm going to close this issue and if you have anything else, email me at support@outred.org
